### PR TITLE
docs(caching): add index examples

### PR DESCRIPTION
**Pull request by AvilaJulio**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 1]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+
+</CodeTabs>
+
+
+You run the following query on a regular basis, with the only difference between queries being the filter values:
+
+```JSON
+{
+  "measures": [
+    "ExampleCube.orderTotal"
```
Comment (pacofvf) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:236]: change to `Orders.orderTotal`

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 2]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+  ],
+  "timeDimensions": [
+    {
+      "dimension": "ExampleCube.timestamp",
```
Comment (pacofvf) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:240]: Change to `Orders.timestamp`.

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 3]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+  ],
+  "order": [
+    [
+      "ExampleCube.timestamp",
+      "asc"
+    ]
+  ],
+  "filters": [
+    {
+      "member": "ExampleCube.productCategory",
```
Comment (pacofvf) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:256]: change to `Orders.productCategory`

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 4]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+      "operator": "equals",
+      "values": [
+        "Dummy1"
+      ]
     },
-  },
+    {
+      "member": "ExampleCube.productName",
```
Comment (pacofvf) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:263]: Change to `Orders.productName`

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 5]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+      "ExampleCube.timestamp",
+      "asc"
+    ]
+  ],
+  "filters": [
+    {
+      "member": "ExampleCube.productCategory",
+      "operator": "equals",
+      "values": [
+        "Dummy1"
```
Comment (pacofvf) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:259]: Change to `Electronics`

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 6]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+      ]
     },
-  },
+    {
+      "member": "ExampleCube.productName",
+      "operator": "equals",
+      "values": [
+        "Dummy2",
+        "Dummy3"
```
Comment (pacofvf) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:267]: Change to :
```
        "Keyboard",
        "Mouse"
```

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 7]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+      ]
+    }
+  ],
+  "dimensions": [
+    "ExampleCube.zipCode"
```
Comment (pacofvf) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:272]: Change to `Orders.zipCode`

**Review by pacofvf — CHANGES_REQUESTED** [review]
Can you change the example query to use the same Cube definition from the other examples in the same page?

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 8]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
-        },
-        aggregatedIndex: {
-          columns: [CUBE.status],
-          type: `aggregate`,
-        },
-      },
+<WarningBox>
+    COLUMN ORDER IN THE INDEX MATTERS
+</WarningBox>
+
```
Comment (hassankhan) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:158]: Might be better to add the following sentence to the paragraph instead?

```suggestion
The order in which columns are specified in the index is **very** important; suboptimal ordering can lead to diminished performance.
```

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 9]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+To improve the performance of an index the main thing to consider is the order of the columns defined in it. The order of the columns should be defined in accordance with the [Index Selectivity](https://blog.toadworld.com/2018/09/05/unselective-indexes-selectivity).
+In a nutshell, index selectivity measures how many unique values a database has. An index is said to have a HIGHER selectivity as the number of unique values goes up and a LOWER selectivity as the number of unique values goes down.
```
Comment (hassankhan) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:160]:
```suggestion
In a nutshell, index selectivity measures how many unique values a database has. An index is said to have a **higher** selectivity as the number of unique values goes up and a **lower** selectivity as the number of unique values goes down.
```

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 10]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+Once we know the selectivity of the columns we'll be adding to the index, we need to place them from **LOWEST** to **HIGHEST** selectivity (this is probably contrary to what you've read about the topic in traditional row-based databases).
```
Comment (hassankhan) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:161]:
```suggestion
Once we know the selectivity of the columns we'll be adding to the index, we need to place them from **lowest** to **highest** selectivity (this is probably contrary to what you've read about the topic in traditional row-based databases).
```

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 11]
```diff
@@ -144,63 +144,300 @@ cube(`Orders`, {
...
+| 2023-01-01 11:00:00 | Plastic Chair | Furniture       | 88524   | 3000       |
+| 2023-01-01 11:00:00 | Keyboard      | Electronics     | 88524   | 2000       |
+
+The pre-aggregation code would look as follows:
+
+<CodeTabs>
+
+``` javascript
+cube('Orders', {
+    sql: `SELECT * FROM Orders;`,
```
Comment (hassankhan) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:181]:
```suggestion
    sql: `SELECT * FROM Orders`,
```

**Review by hassankhan — COMMENTED** [review]
Really great work @AvilaJulio 💯 

Other than the suggestions, one thing we're starting to do in the docs is to use snake-case names in both JavaScript and YAML (for the schema properties, dimensions and measures).

**Review by pacofvf — APPROVED** [review]
LGTM

**Review by hassankhan — APPROVED** [review]

**Pull request merged**